### PR TITLE
fix: Content hidden under action bar

### PR DESCRIPTION
**Pull request by vinnyjth**
**IMPORTANT: If this is for testing code in our npm dependencies, you should be pointing your PR at the `canary` or `next` branch. `master` should never rely on unreleased code.**

### CODE REVIEW

**Comment by vinnyjth** [comment]
blocked by https://github.com/ApollosProject/apollos-apps/pull/1704

(but can be merged before the canary release drops, since it won't break the app without the corresponding client code)

File: apolloschurchapp/src/content-single/ContentSingle.js [hunk 1]
```diff
@@ -9,16 +9,22 @@ import {
...
+const NodeSingleConnectedWithActionContainer = styled(
+  ({ theme: { sizing } }) => ({ paddingBottom: sizing.baseUnit * 5 })
+)(NodeSingleConnected);
+
 const ContentSingle = (props) => {
   const nodeId = props.route?.params?.itemId;
   const { data, loading } = useQuery(
     gql`
-      query getContentNodeTitle($nodeId: ID!) {
+      query getContentNodeTitle($nodeId: ID) {
```
Comment (redreceipt) [inline apolloschurchapp/src/content-single/ContentSingle.js:24]: what's this change for?

Comment (vinnyjth) [reply apolloschurchapp/src/content-single/ContentSingle.js:24]: Ah, I think that's a merge conflict. The change below (id being moved) fixes that query and stops it form erroring (ContentNode doesn't have an ID property)

Comment (vinnyjth) [inline apolloschurchapp/src/content-single/ContentSingle.js:24]:
```suggestion
      query getContentNodeTitle($nodeId: ID!) {
```

File: apolloschurchapp/src/content-single/ContentSingle.js [hunk 2]
```diff
@@ -9,16 +9,22 @@ import {
   ThemeMixinConnected,
 } from '@apollosproject/ui-connected';
 
+import { styled } from '@apollosproject/ui-kit';
+
 import ActionContainer from './ActionContainer';
 
+const NodeSingleConnectedWithActionContainer = styled(
```
Comment (redreceipt) [inline apolloschurchapp/src/content-single/ContentSingle.js:16]:
```suggestion
const PaddedNodeSingleConnected = styled(
```

File: apolloschurchapp/src/content-single/ContentSingle.js [hunk 3]
```diff
@@ -41,9 +47,9 @@ const ContentSingle = (props) => {
           itemId: nodeId,
         }}
       />
-      <NodeSingleConnected nodeId={nodeId}>
+      <NodeSingleConnectedWithActionContainer nodeId={nodeId}>
```
Comment (vinnyjth) [inline apolloschurchapp/src/content-single/ContentSingle.js:50]:
```suggestion
      <PaddedNodeSingleConnected nodeId={nodeId}>
```

File: apolloschurchapp/src/content-single/ContentSingle.js [hunk 4]
```diff
@@ -41,9 +47,9 @@ const ContentSingle = (props) => {
...
         <ActionContainer itemId={nodeId} />
-      </NodeSingleConnected>
+      </NodeSingleConnectedWithActionContainer>
```
Comment (vinnyjth) [inline apolloschurchapp/src/content-single/ContentSingle.js:52]:
```suggestion
      </PaddedNodeSingleConnected>
```

**Review by redreceipt — APPROVED** [review]

**Comment by vinnyjth** [comment]
@redreceipt Ready for a re-review.

**Review by redreceipt — APPROVED** [review]

**Pull request merged**